react/features/toolbox/components/native/RaiseHandButton.js

```diff
@@ -0,0 +1,111 @@
+// @flow
```

Comment:
Since we are ading this, we should put it in a platform agnostic place and make the web use it, since AbstractButton works on both.

Reply:
Very true, I just don't want to push this PR to that extent.